src/frontend/js/utils/test/expectBanner.ts

```diff
@@ -0,0 +1,21 @@
+import { findByText, waitFor } from '@testing-library/react';
+import { BannerType } from 'components/Banner';
+
+export const expectBannerError = async (message: string, rootElement: ParentNode = document) => {
```

Comment:
```suggestion
export const findBannerError = async (message: string, rootElement: ParentNode = document) => {
```

Reply:
I find the term `expect` to match better jest terminology. Hence, we already use this function in various place for some time 😕